### PR TITLE
Issue #144 ContactRouter now sends email according to Parameter ENQUI…

### PR DESCRIPTION
**Pull request by jeremyherr**
ContactRouter now sends email according to Parameter ENQUIRY_EMAILS

 - Added ENQUIRY_EMAILS Parameter to app.py
 - ContactRouter sends an email for each entry in the array
   ENQUIRY_EMAILS.

### CODE REVIEW

File: lux/extensions/smtp/views.py [hunk 1]
```diff
@@ -33,12 +29,16 @@ def post(self, request):
         form = ContactForm(request, data=data)
         if form.is_valid():
             email = request.app.email_backend
-            subject = 'Message from %s' % request.config['APP_NAME']
-            email.send_mail(to=form.cleaned_data['email'],
-                            subject=subject,
-                            message=form.cleaned_data['body'])
-            data = dict(success=True,
-                        message="Message sent")
```
Comment (lsbardel) [inline lux/extensions/smtp/views.py]: Thanks for patch.
I think we want to send an email confirming we have received the message too.
Something from a template.
```
subject: Enquiry received, thank you.
```
and
```
body: We have received your enquiry and we will be incontact with you shortly... and bla bla bla
```

File: lux/core/app.py [hunk 2]
```diff
@@ -225,6 +225,8 @@ class Application(ConsoleParser, Extension, EventMixin):
                   'List of email for site managers'),
         Parameter('EMAIL_BACKEND', 'lux.core.mail.EmailBackend',
                   'Default locale'),
+        Parameter('ENQUIRY_EMAILS', [],
+                  'List of email messages to be sent on reception of enquiry'),
```
Comment (lsbardel) [inline lux/core/app.py]: Can you place this parameter in the ``lux.extensions.smtp`` extension?

File: lux/extensions/smtp/views.py (hunk 1 above)
Comment (jeremyherr) [inline lux/extensions/smtp/views.py]: The code as it is now will send any arbitrary list of emails that the app defines. I've already got both emails you mentioned here: https://github.com/bmlltech/bmll-portal/blob/issue222-enquiry-email/bmllportal/settings.py#L80

**Comment by jeremyherr** [comment]
ENQUIRY_EMAILS parameter has been moved into smtp extension

File: lux/core/app.py (hunk 2 above)
Comment (jeremyherr) [inline lux/core/app.py]: yes, done

**Pull request merged**